### PR TITLE
Home Page

### PR DESCRIPTION
**Pull request by kjniemela**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-# archivium
\ No newline at end of file
+# Archivium
+
+Archivium is a worldbuilding tool, allowing you to organize everything about your world in one place.
```
Comment (JoelNiemela) [inline README.md:3]: If FMAT is planning to use this, do we want to update this description right away? Not sure.

File: api/models/item.js [hunk 2]
```diff
@@ -68,7 +70,8 @@ async function getMany(user, conditions, permissionsRequired=perms.READ, basicOn
         item.universe_id,
         user.username as author,
         universe.title as universe,
-        ${selectString}
+        universe.shortname as universe_short,
+        ${selectString /* Previous line is in conflict with #23 and must be resolved when that is merged TODO */}
```
Comment (JoelNiemela) [inline api/models/item.js:74]: Reminder to take a look at this TODO comment.

**Review by JoelNiemela — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kjniemela) [reply README.md:3]: What would we put instead?

Comment (JoelNiemela) [reply README.md:3]: Not sure. This is def. not a blocker if we can't think of anything.

Comment (JoelNiemela) [reply README.md:3]: Actually, I didn't realize this is only a README (I thought it was rendered somewhere) — this is probably the correct README description.

Comment (kjniemela) [reply README.md:3]: Okay. I'll merge this, then. I can't think of anything better either - and really, worldbuilding _is_ what this is meant for, anyway.

**Pull request merged**